st2common/st2common/validators/api/reactor.py

```diff
@@ -15,7 +15,11 @@
 
 import six
 
+from apscheduler.triggers.cron import CronTrigger
```

Comment:
this is pretty bad I am not sure it is even work breaking the abstraction to provide the validation.